groups/xml-infrastructure/templates/bep_cron.tpl

```diff
@@ -1,93 +1,108 @@
+#LIVE#*/1 20-23 * * 0 /home/ewfbeplive/efbackend/formPartition.sh >/dev/null 2>&1
+#LIVE
+#LIVE#########################LIVE SERVICE PROCESSING END#########################
+#LIVE
+#LIVE#########################FormProcess#########################################
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formProcess.sh General >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formProcess.sh Incorporations >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formProcess.sh SameDayIncorporations >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formProcess.sh Shuttles >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formProcess.sh Accounts >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formProcess.sh Mortgages >/dev/null 2>&1
+#LIVE#########################OTHER SCHEDULED JOBS################################
+#LIVE*/1 0-23 * * * /home/xml/efbackend/CreateCerts.sh >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/formResponse.sh >/dev/null 2>&1
+#LIVE*/1 0-23 * * * /home/xml/efbackend/emailDispatcher.sh >/dev/null 2>&1
+#LIVE00 4 * * * /home/xml/efbackend/corporationTax.sh >/dev/null 2>&1
+#LIVE###email poll reminders every morning at 4-30am
+#LIVE30 4 * * * /home/xml/efbackend/emailPollReminder.sh >/dev/null 2>&1
+#LIVE###email unpolled submission report every Monday Morning at 5-30am
+#LIVE10 20 * * * /home/xml/efbackend/efimport.sh >/dev/null 2>&1
+#LIVE59 23 * * * /home/xml/efbackend/getFilingStats.pl xml >/dev/null 2>&1
+#LIVE3 * * * * /home/xml/efbackend/xmlDigestCacheDelete.sh >/dev/null 2>&1
+#LIVE#ensure weeding runs AFTER archiving
+#LIVE14,44 * * * * /home/xml/efbackend/formArchive.sh 365 >/dev/null 2>&1
+#LIVE6,36 * * * * /home/xml/efbackend/weed.sh xml >/dev/null 2>&1
+#LIVE#delete the xml tif and txt files
+#LIVE02 02 * * * find /mnt/filling/image/xmlgw -type f -mtime +10  -exec rm -f {} ; >/dev/null 2>&1
+#LIVE######################################SUPPORT SECTION START##############################
+#LIVE*/1 * * * * /home/xmlbeplive/checkapache.sh >/dev/null 2>&1
+#LIVE0 11,16 * * * /home/xml/efbackend/produceStats.sh SupportHourlyStatsReport "sbasile@companieshouse.gov.uk,tjohn@companieshouse.gov.uk,mwilliams5@companieshouse.gov.uk,dcornelius@companieshouse.gov.uk,678104@gmail.com" >/dev/null 2>&1
+#LIVE*/1 * * * * /home/xml/errorlogcheck.sh -700 >/dev/null 2>&1
+#LIVE*/10 0-23 * * * /home/xml/efbackend/monitorLCKfilesWarn.sh >/dev/null 2>&1
+#LIVE*/1 * * * * /home/xml/checkformprocess.sh >/dev/null 2>&1
+#LIVE4,34 * * * * /home/xml/efbackend/filinqueuecheck.sh email 100 >/dev/null 2>&1
+#LIVE0 8-17 * * * /home/xmlbeplive/updateXMLFormDetail.sh >/dev/null 2>&1
+#LIVE0 10 * * * /home/xmlbeplive/failedCerts.sh >/dev/null 2>&1
+#LIVE05 10 * * * /home/xmlbeplive/failedMortCerts.sh >/dev/null 2>&1
+#LIVE*/15 7-18 * * * /home/xmlbeplive/efbackend/status7TNEP.sh >/dev/null 2>&1
+#LIVE*/15 7-18 * * * /home/xmlbeplive/efbackend/updateGenBarcodeSub.sh >/dev/null 2>&1
```

Comment:
Just noticed the home directory is incorrect here. Should be `/home/xml`. 
There are a few more in the file as well.

Reply:
just updated